### PR TITLE
Doc 2.0

### PR DESCRIPTION
**Pull request by KitaitiMakoto**

### CODE REVIEW

**Comment by KitaitiMakoto** [comment]
@fbernier I Added changelog but I'm not sure it's easy to understand. Can you review it?

File: README.md [hunk 1]
```diff
@@ -40,33 +39,21 @@ Tomlrb.load_file('my_file', symbolize_keys: true)
...
-# MEMORY
-Calculating -------------------------------------
-      emancu/toml-rb     2.733M memsize (    40.000  retained)
-                        35.718k objects (     1.000  retained)
-                        50.000  strings (     1.000  retained)
-     fbernier/tomlrb   115.778k memsize (    80.000  retained)
-                         2.422k objects (     2.000  retained)
-                        50.000  strings (     2.000  retained)
+      emancu/toml-rb     10.630  (±28.2%) i/s -     49.000  in   5.035997s
+     fbernier/tomlrb    181.585  (±24.8%) i/s -    850.000  in   5.019273s
```
Comment (fbernier) [inline README.md:52]: Unsurprisingly all these new features are quite heavy :/

**Comment by fbernier** [comment]
@KitaitiMakoto I pushed some tiny changes but it was pretty good. Thanks!

File: README.md (hunk 1 above)
Comment (KitaitiMakoto) [reply README.md:52]: Yes...

**Comment by KitaitiMakoto** [comment]
@fbernier  Thank you for the review! We're ready to release. You gave me a right to push to rubygems.org. Can I do that myself?

**Comment by fbernier** [comment]
If you feel confident, go ahead. Let me know if you have any questions. Thanks again for this work!

**Comment by KitaitiMakoto** [comment]
Okay, I will try!

**Comment by KitaitiMakoto** [comment]
Done!

**Pull request merged**